src/e2e-test/java/io/cdap/cdap/ui/stepsdesign/CommonSteps.java

```diff
@@ -56,7 +56,7 @@ public void openPipelineStudioPage() {
       }
     }
     // wait for rendering to finish otherwise elements are not attached to dom
-    Helper.waitSeconds(3);
```

Comment:
hmm i see we just keep incrementing the seconds from past commits. seems bad. Maybe we can do something like wait till bq source plugin to display. But it's ok for now

Reply:
I agree, tbf I went with 4 in the refactoring PR initially. but if it comes to increasing it once more, will make a diff change something like you suggested.